.github/workflows/application-signals-python-e2e-ec2-test.yml

```
@@ -40,7 +40,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           repository: ${{ inputs.caller-workflow-name == 'main-build' && 'aws-observability/aws-application-signals-test-framework' || github.repository }}
-          ref: ${{ inputs.caller-workflow-name == 'main-build' && 'adot-pending-release' || github.ref }}
+          ref: ${{ github.ref }}
```

Same issue as EKS, should be the following to account for release testing:
```suggestion
          ref: ${{ inputs.caller-workflow-name == 'main-build' && 'main' || github.ref }}
```

---

Ok added to ec2 too